coordinator/src/orderbook/db/custom_types.rs

```diff
@@ -130,6 +131,51 @@ impl FromSql<OrderStateType, Pg> for OrderState {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, FromSqlRow, AsExpression)]
+#[diesel(sql_type = OrderReasonType)]
+pub(crate) enum OrderReason {
+    Manual,
```

Comment:
What's an `manual` order? 

Maybe add a description to what each variant means please?